src/systempolicy/systemPolicy.go

```diff
@@ -772,8 +772,7 @@ func mergeSysPolicies(pols []types.KnoxSystemPolicy) []types.KnoxSystemPolicy {
 	var results []types.KnoxSystemPolicy
 	for _, pol := range pols {
 		pol.Metadata["name"] = "autopol-system-" +
-			strconv.FormatUint(uint64(common.HashInt(pol.Metadata["labels"]+
-				pol.Metadata["namespace"]+pol.Metadata["clustername"]+pol.Metadata["containername"])), 10)
+			strconv.FormatUint(uint64(common.HashInt(pol.Metadata["labels"]+pol.Metadata["namespace"]+pol.Metadata["clustername"])), 10)
```

Comment:
if we use labels, what if the following scenario occurs
1. Log has pod1 --> labels (app=web,component=frontend,env=testing)
2. Log has pod1 --> labels (env=testing,component=frontend,app=web)
This will create a new policy name right?

Reply:
@vishnusomank While forming the labels, we are sorting the labels and storing it.